pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -5,6 +5,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	kubeletconfigv1beta1 "k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig/v1beta1"
```

Comment:
why would i not create a file with these configuration if that's what is getting exposed in this new api?

Reply:
How would you create an instance of a newer file? I couldn't find any documentation on it.